### PR TITLE
CRIT Expansion update

### PR DESCRIPTION
**Pull request by Soupster89**
#### Summary
```SUMMARY: Mods "Balance of items and description changes"```

#### Purpose of change
New content:

3 new guns.

Quality of Life:
Items now have C.R.I.T condensed down to much easier-to-look-at CRIT to reduce inventory UI vomiting on your eyes.

Balance:
Various weapons have lost artefact qualities as those were honestly quite unnecessary and have reduced damage to compensate for various techniques.

Tool-Armor such as CRIT EM Vest and Gasmask require more energy and no longer buff some stats, they also overheat as they should but also a little less encumbrance.

Martial arts are less overbearing, more deviation between styles as well. CQB is now a hand to hand (unless you have a fork) and Bladework is for knives. Very fun. Maybe too strong?

Some classes were changes. Does one class _really_ need all good bionics? No.

Some items had wrong materials, now corrected. This helps keep armor in line.

Remaining Changes:
- [x] Fix Vampire Mutation line. It does not need everything! 

- [x] Add more to Wendigo mutations, very lacking.

- [x] Correct Mutations to current JSON, - some JSON-ized effects are missing

- [x] Fix more classes, very lackluster or overly bloated

- [x] Introduce monsters

- [x] Add some item spawns and more monster drops

- [x] Rebalance Enforcement melee style

- [x]  Tweak various guns

- [x] Tweak armor

- [x]  Lint all the things

### CODE REVIEW

**Comment by LaVeyanFiend** [comment]
Before you take off the WIP, you need to add ``"melee_allowed: true"`` to your C.R.I.T. CQB/Enforcement/Bladework techniques, as currently none can be used with the weapons you allowed for each style.

**Comment by LaVeyanFiend** [comment]
jenkins rebuild

**Comment by anothersimulacrum** [comment]
The `[Mods]` in your summary line should be `Mods`

**Comment by minhmap859** [comment]
Please add "weight_capacity_modifier": 1.35 to the Strong Back perk in both crt_mutation and crt_vamp_mutations.

**Comment by LaVeyanFiend** [comment]
jenkins rebuild

File: data/mods/CRT_EXPANSION/mutations/crt_wendigo_mutations.json [hunk 1]
```diff
@@ -9,6 +9,61 @@
     "purifiable": false,
     "threshold": true
   },
+    {
```
Comment (AMurkin) [inline data/mods/CRT_EXPANSION/mutations/crt_wendigo_mutations.json]:
```suggestion
  {
```

**Comment by ZhilkinSerg** [comment]
Is it done yet?

File: data/mods/CRT_EXPANSION/martial/CRT_Bladework.json [hunk 2]
```diff
@@ -96,10 +98,10 @@
     "min_unarmed": 2,
     "min_melee": 2,
     "unarmed_allowed": true,
+    "melee_allowed": true,
     "mult_bonuses": [ [ "movecost", 0.65 ], [ "damage", "bash", 0.55 ], [ "damage", "cut", 0.6 ], [ "damage", "stab", 0.55 ] ],
     "flat_bonuses": [ [ "damage", "stab", "dex", 0.01 ], [ "damage", "cut", "dex", 0.015 ], [ "movecost", "dex", -0.45 ] ],
     "messages": [ "You swiftly jab your weapon into %s joints", "<npcname> swiftly jabs their weapon into %s" ],
```
Comment (ZhilkinSerg) [inline data/mods/CRT_EXPANSION/martial/CRT_Bladework.json]:
```suggestion
    "messages": [ "You swiftly jab your weapon into %s joints", "<npcname> swiftly jabs their weapon into %s" ]
```

File: data/mods/CRT_EXPANSION/martial/CRT_Bladework.json [hunk 3]
```diff
@@ -84,10 +86,10 @@
     "min_unarmed": 1,
     "min_melee": 1,
     "unarmed_allowed": true,
+    "melee_allowed": true,
     "mult_bonuses": [ [ "damage", "bash", 0.8 ], [ "damage", "cut", 0.7 ], [ "damage", "stab", 0.6 ] ],
     "flat_bonuses": [ [ "damage", "stab", "dex", 0.15 ], [ "damage", "cut", "dex", 0.2 ], [ "movecost", "dex", -0.5 ] ],
     "messages": [ "You release a blinding slash at %s", "<npcname> slashes at %s" ],
```
Comment (ZhilkinSerg) [inline data/mods/CRT_EXPANSION/martial/CRT_Bladework.json]:
```suggestion
    "messages": [ "You release a blinding slash at %s", "<npcname> slashes at %s" ]
```

**Comment by Soupster89** [comment]
> Is it done yet?

Sadly it has not been finished yet.

**Comment by Soupster89** [comment]
I've ran crt_wendigo_mutations.json through the linter and I keep getting errors with Travis, what should I do?

**Comment by anothersimulacrum** [comment]
Travis isn't having issues with that file, it's having issues with crt_gun.json.

**Comment by LaVeyanFiend** [comment]
@Soupster89 , are you dead? 25 days ago since your last edit to this?

**Comment by Soupster89** [comment]
I'm back. I've had many personal issues come up in the last month and now that I don't really have anyone else to deal with, I have quite a bit of free time, other than my job.

**Comment by Soupster89** [comment]
How do I extend scenarios without breaking them?

It seems I have only my CRIT classes in the heli crash and wilderness scenarios.
 
A recent pr also references this.

**Comment by ZhilkinSerg** [comment]
Use https://github.com/CleverRaven/Cataclysm-DDA/pull/32612 as a reference.

**Comment by Soupster89** [comment]
> Use #32612 as a reference.

It is to my understanding that I am to replace the archaic "extend" node with "add_professions" node?

**Comment by ZhilkinSerg** [comment]
Someone already fixing it in https://github.com/CleverRaven/Cataclysm-DDA/pull/32949.

**Comment by Soupster89** [comment]
What file is it having issues with? 

Bladework or dragonslayer?

**Comment by Soupster89** [comment]
What is this having an error on?

It shows that it has issues with "CRT_Bladework" and "CRT_Melee_Buffs".

But at the same time?

**Comment by ZhilkinSerg** [comment]
You need to [lint](http://dev.narc.ro/cataclysm/format.html) your json files.

Multiple errors are probably because linter on Travis CI is run on multiple threads in parallel.

**Comment by Soupster89** [comment]
Thank you

**Comment by Soupster89** [comment]
All good.

**Comment by ZhilkinSerg** [comment]
I well test tonight.

**Comment by Soupster89** [comment]
Neat

**Pull request merged**